README.md

```diff
@@ -209,16 +176,26 @@ and improve:
 ![Architecture](docs/img/modin_architecture.png)
 
 Visit the [Documentation](https://modin.readthedocs.io/en/latest/development/architecture.html) for
-more information, and checkout [the difference between Modin and Dask!](https://github.com/modin-project/modin/tree/master/docs/modin_vs_dask.md)
+more information!
 
 **`modin.pandas` is currently under active development. Requests and contributions are welcome!**
 
 
-### More information and Getting Involved
+### General Resources and Getting Involved
+
+- [Documentation Link](https://modin.readthedocs.io/en/latest/)
+- [Modin Examples + Tutorials](https://modin.readthedocs.io/en/latest/getting_started/examples.html)
+- [Modin Talks + Blogs](https://modin.readthedocs.io/en/latest/getting_started/examples.html#talks-podcasts)
+- [Modin Slack](https://join.slack.com/t/modin-project/shared_invite/zt-yvk5hr3b-f08p_ulbuRWsAfg9rMY3uA)
+- [Modin Discourse](https://discuss.modin.org)
+- [Modin GitHub Issues Page](https://github.com/modin-project/modin/issues)
+- [Modin FAQs](https://modin.readthedocs.io/en/latest/getting_started/faq.html)
+- [Modin Troubleshooting Guide](https://modin.readthedocs.io/en/latest/getting_started/troubleshooting.html)
+- [Modin Paper](https://people.eecs.berkeley.edu/~totemtang/paper/Modin.pdf)
```

Comment:
This feels very list-ey and if I was a user, I wouldn't want to click on any of these when I skim over this. I'll try to think if there's a better organization. Thoughts welcome!

Reply:
Yeah, I was modeling off of the Tensorflow README (https://github.com/tensorflow/tensorflow) and PyTorch does something similar but hopefully the list is short enough that people will find it easy enough to browse. +1 to anyone having ideas :)

Reply:
Thanks @naren-ponder, the tensorflow and pytorch links are really helpful. What about something like this? I added a few links and reorganized them into sections.

```
### Other Resources

#### Getting Started with Modin

- [Documentation](https://modin.readthedocs.io/en/latest/)
- [10-min Quickstart Guide](https://modin.readthedocs.io/en/latest/getting_started/quickstart.html)
- [Examples and Tutorials](https://modin.readthedocs.io/en/latest/getting_started/examples.html)
- [Videos and Blogposts](https://modin.readthedocs.io/en/latest/getting_started/examples.html#talks-podcasts)

#### Modin Community

- [Slack](https://modin.org/slack.html)
- [Discourse](https://discuss.modin.org)
- [Twitter](https://twitter.com/modin_project)
- [Mailing List](https://groups.google.com/g/modin-dev)
- [GitHub Issues](https://github.com/modin-project/modin/issues)
- [StackOverflow](https://stackoverflow.com/questions/tagged/modin)

#### Learn More about Modin

- [Frequently Asked Questions (FAQs)](https://modin.readthedocs.io/en/latest/getting_started/faq.html)
- [Troubleshooting Guide](https://modin.readthedocs.io/en/latest/getting_started/troubleshooting.html)
- [Development Guide](https://modin.readthedocs.io/en/latest/development/index.html)
- Modin is built on many years of research and development at UC Berkeley. Check out these selected paper to learn more about how Modin works:
  - [Flexible Rule-Based Decomposition and Metadata Independence in Modin](https://people.eecs.berkeley.edu/~totemtang/paper/Modin.pdf) (VLDB 2021)
  - [Dataframe Systems: Theory, Architecture, and Implementation](https://www2.eecs.berkeley.edu/Pubs/TechRpts/2021/EECS-2021-193.pdf) (PhD Dissertation 2021)
  - [Towards Scalable Dataframe Systems](https://arxiv.org/pdf/2001.00888.pdf) (VLDB 2020)

#### Getting Involved

***`modin.pandas` is currently under active development. Requests and contributions are welcome!***

For more information on how to contribute to Modin, check out the
[Modin Contribution Guide](https://modin.readthedocs.io/en/latest/development/contributing.html).
```

Reply:
@dorisjlee This looks awesome!! Thanks for the suggestion :)